pulsar/README.md

```diff
@@ -0,0 +1,47 @@
+# Apache Pulsar Initialization Action
+
+Here we've got an initialization action that automates the installation of a single-cluster pulsar instance on bare metal. More detail can be found in the Pulsar documentation here: https://pulsar.apache.org/docs/en/deploy-bare-metal/
+
+## Usage
+
+Specify the .sh script's location when using gcloud to create your dataproc cluster. Pulsar requires Zookeeper, which can be installed by either including it as an optional component (https://cloud.google.com/dataproc/docs/concepts/components/zookeeper), or by creating a HA cluster by specifying --num-masters=3.
+
+e.g.
+```bash
+gcloud dataproc clusters create cluster-name \
+    --region=region \
+    --initialization-actions=gs://init_action_storage/pulsar-initialization-action.sh \
+    --optional-components=ZOOKEEPER \
+    --metadata=pulsar-functions-enabled=true,builtin-connectors-enabled=true,tiered-storage-offloaders-enabled=true... \
```

Comment:
You need to use multiple `--metadata` flags, i.e., `--metadata=pulsar-functions-enabled=true --metadata=builtin-connectors-enabled=true ...`.

Reply:
Is this accurate? In the Google docs they use a single --metadata flag w/ comma separated metadata values. Please correct me if I'm wrong here. See here: https://cloud.google.com/dataproc/docs/concepts/configuring-clusters/metadata